bitrise.yml

```diff
@@ -1436,7 +1437,7 @@ workflows:
             DERIVED_DATA_DIR="/Users/vagrant/git/DerivedData"
             PROJECT="/Users/vagrant/git/firefox-ios/Client.xcodeproj"
             SCHEME="Fennec_Enterprise"
-            SDK="iphoneos"
+            SDK="iphoneos18.2"
```

Comment:
Can this come from any other build environment variable so that in the future it doesn't need to be updated?

Reply:
No, the setting needs to be "iphoneos18.2" and not just "iphoneos". I tried just "iphoneos" and `xcodebuild` said no such SDK available. Here's what `xcodebuild` told me about the SDK available:
```
$ xcodebuild -showsdks | grep -E 'iphoneos|iphonesimulator'
        iOS 18.2                        -sdk iphoneos18.2
        Simulator - iOS 18.2            -sdk iphonesimulator18.2
```